lib/class/song.class.php

```diff
@@ -588,8 +593,8 @@ private function has_info($limit_threshold = '')
 
         $sql = 'SELECT `song`.`id`, `song`.`file`, `song`.`catalog`, `song`.`album`, `album`.`album_artist` AS `albumartist`, `song`.`year`, `song`.`artist`, ' .
             '`song`.`title`, `song`.`bitrate`, `song`.`rate`, `song`.`mode`, `song`.`size`, `song`.`time`, `song`.`track`, ' .
-            '`song`.`played`, `song`.`enabled`, `song`.`update_time`, `song`.`mbid`, `song`.`addition_time`, `song`.`license`, ' .
-            '`song`.`composer`, `song`.`user_upload`, `album`.`disk`, `album`.`mbid` AS `album_mbid`, `artist`.`mbid` AS `artist_mbid`, `album_artist`.`mbid` AS `albumartist_mbid` ' .
+            '`song`.`played`, `song`.`enabled`, `song`.`update_time`, `song`.`mbid`, `song`.`addition_time`, `song`.`license`, `song`.`url_source`, `song`.`url_publisher`, `song`.`url_file`, ' .
```

Comment:
reverted this file, not sure what all these are from but it exploded with errors

Reply:
Ouch!  The branch I created the PR on must have been created from the current feature I was working on and not from the develop branch. I had manually added those columns to the db while I was working on the feature.

Thanks